community/kernel/src/main/java/org/neo4j/unsafe/batchinsert/BatchInserterImpl.java

```diff
@@ -285,7 +284,8 @@ public Label apply( long from )
         Token[] types = getRelationshipTypeStore().getTokens( Integer.MAX_VALUE );
         relationshipTypeTokens = new BatchTokenHolder( types );
         indexStore = life.add( new IndexConfigStore( this.storeDir, fileSystem ) );
-        schemaCache = new SchemaCache( neoStore.getSchemaStore() );
+        // TODO: perhaps we should differentiate for BatchInserter running in enterprise here...
+        schemaCache = new SchemaCache( new StandardConstraintSemantics(), neoStore.getSchemaStore() );
```

Comment:
Should we support creating PEC in BatchInserter?
If we should, then we need an EnterpriseModule here to get access to the EnterpriseConstraintSemantics, otherwise the "TODO" can be removed.